### PR TITLE
access to raw SAM format header text

### PR DESCRIPTION
**Pull request by pmarks**

### CODE REVIEW

**Comment by pmarks** [comment]
@johanneskoester @christopher-schroeder - can you guys take a look?

File: src/bam/mod.rs [hunk 1]
```diff
@@ -507,9 +506,9 @@ impl HeaderView {
     }
 
     /// Retrieve the textual SAM header as a String
-    pub fn text(&self) -> Result<String, FromUtf8Error> {
+    pub fn text(&self) -> Vec<u8> {
```
Comment (johanneskoester) [inline src/bam/mod.rs]: What about just returning the byte slice here? I don't think we need the reallocation, do we?

**Comment by pmarks** [comment]
@johanneskoester - yes, good call

**Comment by johanneskoester** [comment]
Great, thank you! Sorry for all the nitpicking :-).

**Pull request merged**